applications/callflow/src/module/cf_dynamic_cid.erl

```diff
@@ -2,13 +2,17 @@
 %%% @copyright (C) 2011-2014, 2600Hz INC
 %%% @doc
 %%% "data":{
+%%%   "action": "manual" | "list"
 %%%   "media_id":"id_of_media"
 %%%   // optional after this
 %%%   "interdigit_timeout":2000
+%%%   "id":"01fc6_change_me_294234d2324497" // the list referenced above is kept in this document
```

Comment:
We would use `{LIST_ID}` or similar as a placeholder for data instead of garbled text.